cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -192,7 +194,7 @@ def start_master(etcd, tls):
         hookenv.log('Starting {0} service.'.format(service))
         host.service_start(service)
     hookenv.open_port(6443)
-    hookenv.status_set('active', 'Kubernetes master services ready.')
+    hookenv.log('Kubernetes master services ready.')
```

Comment:
I left it in as a log message to be cautious, but I think you're right, we don't really gain anything from it. I'll remove it!